src/nplinker/genomics/bgc.py

```diff
@@ -173,6 +174,36 @@ def is_mibig(self) -> bool:
         """
         return self.id.startswith("BGC")
 
+    def to_dict(self) -> dict[str, Any]:
+        """Convert the BGC object to a dictionary for exporting results.
+
+        Compiles relevant information from the BGC object and formats it into a dictionary.
+        Each key-value pair in the dictionary represents a specific attribute of the BGC.
+
+        Returns:
+            A dictionary containing the following key-value pairs:
+            - GCF_id (list[str]): A list of GCF IDs.
+            - GCF_bigscape_class (list[str | None]): A list of BiG-SCAPE classes.
```

Comment:
```suggestion
            - GCF_bigscape_class (list[str]): A list of BiG-SCAPE classes.
```
Please check if it's what you want.